tests/test_qiskit.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import sys
+import numpy as np
+from yaml import parse
```

Comment:
`parse` is imported but not used? The test fails because it is not installed. We can include it if necessary but it does not seem like the case?

Reply:
That was included accidently. Removed it now.